src/cache/cache_las.c

```diff
@@ -908,8 +910,8 @@ __las_sweep_count(WT_CACHE *cache)
 	 * with lookaside entries are blocked during sweep, make sure we do
 	 * some work but don't block reads for too long.
 	 */
-	return ((uint64_t)WT_MAX(100,
-	    cache->las_entry_count / (WT_MINUTE * 5 / WT_LAS_SWEEP_SEC)));
+	return ((uint64_t)WT_MAX(100, __wt_las_entry_count(cache) /
+	     (5 * WT_MINUTE / WT_LAS_SWEEP_SEC)));
```

Comment:
`WT_MAX` evaluates its arguments twice, so `__wt_las_entry_count()` is potentially called twice. It's not obvious to me that's a problem, but I'd suggest fixing it to remove any question.